webrtc.html

```diff
@@ -4768,7 +4768,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             nullable</dt>
             <dd>
               <p>The <code>track</code> attribute is the track that is
-              associated with this <code><a>RTCRtpSender</a></code> object.</p>
+              associated with this <code><a>RTCRtpSender</a></code> object.
+              If <code>track.stop()</code> is called, or if 
```

Comment:
What if the track is muted and VAD is off?  The purpose of turning VAD off is to make sure we keep sending RTP.

Perhaps instead we should say "an ended or muted is treated the same as a null track" (addTransceiver that has never had a track, for example).